includes/admin/meta-boxes/class-wc-meta-box-product-images.php

```diff
@@ -49,13 +49,18 @@ public static function output( $post ) {
 							$update_meta = true;
 							continue;
 						}
-
-						echo '<li class="image" data-attachment_id="' . esc_attr( $attachment_id ) . '">
-								' . $attachment . '
-								<ul class="actions">
-									<li><a href="#" class="delete tips" data-tip="' . esc_attr__( 'Delete image', 'woocommerce' ) . '">' . __( 'Delete', 'woocommerce' ) . '</a></li>
-								</ul>
-							</li>';
+						?>
+						<li class="image" data-attachment_id="<?php echo (int) $attachment_id; ?>">
```

Comment:
We should keep using `esc_attr()`, please do not revert it.

Reply:
Done, please check.